### PR TITLE
doc: added object detection demo in python

### PR DESCRIPTION
**Pull request by christyjacob4**
Added a demo to leverage cloud functions and detect objects in an uploaded image

### CODE REVIEW

File: python/object-detection/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```bash
+$ cd ..
+$ tar -zcvf code.tar.gz object-detection
+```
+
+* Upload the tarfile to your Appwrite Console and use the following entrypoint command
+
+```bash
+python main.py
+```
```
Comment (eldadfux) [inline python/object-detection/README.md:39]: Also need to add a listener for file events (storage.files.create, storage.files.update, storage.files.delete).

File: python/object-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# 📷 Object Detection using Cloudmersive Vision API
+A sample Python Cloud Function for object detection on an image file uploaded by the user. 
+
+## 📝 Environment Variables
+Add the following environment variables in your Cloud Functions settings.
+
+* **APPWRITE_KEY** - Create a key from the Appwrite console with the following scopes (`files.read` and `files.write`)
```
Comment (eldadfux) [inline python/object-detection/README.md:7]: I think read scope should be enough

File: python/object-detection/main.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# create an instance of the API class
+api_instance = cloudmersive_image_api_client.RecognizeApi(cloudmersive_image_api_client.ApiClient(configuration))
+image_file = FILENAME # file | Image file to perform the operation on.  Common file formats such as PNG, JPEG are supported.
+
+try:
+    # Detect objects including types and locations in an image
+    api_response = api_instance.recognize_detect_objects(image_file)
+    pprint(api_response)
+except ApiException as e:
+    print("Exception when calling RecognizeApi->recognize_detect_objects: %s\n" % e)
```
Comment (eldadfux) [inline python/object-detection/main.py:48]: Code looks awesome, clean & elegant ❤️

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: python/object-detection/README.md (hunk 1 above)
Comment (christyjacob4) [reply python/object-detection/README.md:39]: Done

File: python/object-detection/README.md (hunk 2 above)
Comment (christyjacob4) [reply python/object-detection/README.md:7]: Done

File: python/object-detection/main.py (hunk 3 above)
Comment (christyjacob4) [reply python/object-detection/main.py:48]: Thanks ❤️  👍

**Review by eldadfux — APPROVED** [review]

**Pull request merged**